problem-based-packs/insecure-transport/js-node/http-request.js

```diff
@@ -0,0 +1,78 @@
+function bad_http() {
+    // ruleid: http-request
+    http.get('http://nodejs.org/dist/index.json', (res) => {
+    const { statusCode } = res;})
+
+    // ruleid: http-request
+    https.get('http://nodejs.org/dist/index.json', (res) => {
```

Comment:
Does this actually go over HTTP? 😱

Reply:
you're totally right!! I don't think it actually goes over http :o

Reply:
We could test using wireshark 🤔   🦈

Reply:
thats a good idea!!

Reply:
I'll first push this, if requests to 'http...' through the https module actually doesn't encrypt traffic, i'll update it back to the OG 💯

Reply:
tested -- node actually throws "TypeError [ERR_INVALID_PROTOCOL]: Protocol "http:" not supported. Expected "https:" error if you attempt to request an http website.